scripts/nginx/readarr.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
+# Nginx conf for Radarr v3
+# Flying sausages 2020
+# Refactored by Bakerboy448 2021
+master=$(cut -d: -f1 < /root/.master.info)
+app_name="readarr"
+if ! READARR_OWNER="$(swizdb get $app_name/owner)"; then
+    READARR_OWNER=$(_get_master_username)
+else
+    READARR_OWNER="$(swizdb get $app_name/owner)"
+fi
+app_port="8787"
+user="$READARR_OWNER"
+app_servicename="${app_name}"
+app_configdir="/home/$user/.config/${app_name^}"
+app_baseurl=$app_name
+
+cat > /etc/nginx/apps/$app_name.conf << READARR
+location /$app_name {
+  proxy_pass        http://127.0.0.1:$app_port/;
+  proxy_set_header Host \$proxy_host;
+  proxy_set_header X-Forwarded-For \$proxy_add_x_forwarded_for;
+  proxy_set_header X-Forwarded-Proto \$scheme;
+  proxy_redirect off;
+  auth_basic "What's the password?";
+  auth_basic_user_file /etc/htpasswd.d/htpasswd.${master};
```

Comment:
I don't see `$master` declared anywhere. You need `$user` I suppose. Test your PRs...
```suggestion
  auth_basic_user_file /etc/htpasswd.d/htpasswd.${user};
```

Reply:
It was tested.

please re-review with coffee in hand.
Master is set in line 5